### DISCUSSION TITLE
OKD 4.7 installation failed

### DISCUSSION THREAD

**Discussion by EmanueleMiz** [okd-project/okd#1027] [General]
**Describe the bug**
The Bootstrap machine failed to deploy the primary machine,
and the worker machines do not start the installation.
plus i never see any files in api-int.gzt-cluster01.gzt.local:22623/config/worker
instead I can see that of the masters api-int.gzt-cluster01.gzt.local:22623/config/master

You can tell me where I'm wrong ?

**Version**
Platform : vSphere
openshift-install-linux-4.7.0-0.okd-2021-09-19-013247

**Log bundle**
[log-bundle-20211217152904.tar.gz](https://github.com/openshift/okd/files/7735895/log-bundle-20211217152904.tar.gz)

**Comment by vrutkovs** [comment]
Is it reproducible on latest stable?

**Comment by EmanueleMiz** [comment]
I have start installation whit last 4.7 version.
I don't know which is the stable version ?

**Reply by vrutkovs** [reply]
See https://github.com/openshift/okd/releases and https://amd64.origin.releases.ci.openshift.org/#4-stable

**Reply by EmanueleMiz** [reply]
OK thank you,
it's a stable version you can't tell me more from the bundle log why the master machines don't install correctly ?

**Reply by vrutkovs** [reply]
Its a stable version, but its not the latest stable version. So first I wonder if we need to apply a fix - or this issue has been resolved in the latest stable?

**Reply by EmanueleMiz** [reply]
ok now I will restart the installation with the latest stable version.
From what I could understand from the log it looks like an error on the resolv.conf of the machines.
This problem has already occurred on the resolv.conf file ?
Do you have any information?
